README.md

```diff
@@ -123,11 +123,11 @@ gcloud container clusters delete onlineboutique \
     --project=${PROJECT_ID} --zone=${ZONE}
 ```
 
-## Provision the GKE cluster and deploy the Online Boutique apps with Terraform
+## Use Terraform to provision a GKE cluster and deploy Online Boutique
 
-Instructions for using Terraform to replicate the [**Quickstart (GKE)**](#quickstart-gke) steps can be found [here](terraform).
+The [`/terraform` folder](terraform) contains instructions for using [Terraform](https://www.terraform.io/intro) to replicate the steps from [**Quickstart (GKE)**](#quickstart-gke) above.
 
-## Other provisioning Options
+## Other deployment variations
```

Comment:
**My rationale for change:**

1. "Provision" usually refers to the process of setting up ([source](https://www.redhat.com/en/topics/automation/what-is-provisioning)). "Deployment" is more holistic (refers to setting up both infrastructure and app/services).

2. Let's stick to one word: "variation" for consistency and clarity. Let's avoid "variant" and "option".